tests/integration/timeout_tests.py

```diff
@@ -23,18 +27,39 @@ def test_default_timeout():
 
 
 def test_route_specific_timeout():
-    baseplate = Baseplate()
+    baseplate = _create_baseplate_object("5 milliseconds")
+    context = baseplate.make_context_object()
+    with baseplate.make_server_span(context, "test"):
+        with pytest.raises(ServerTimeout):
+            gevent.sleep(1)
 
-    observer = TimeoutBaseplateObserver.from_config(
-        {"server_timeout.default": "1 hour", "server_timeout.by_endpoint.test": "5 milliseconds"}
-    )
-    baseplate.register(observer)
+    context = baseplate.make_context_object()
+    with baseplate.make_server_span(context, "test"):
+        gevent.sleep(0)  #
```

Comment:
what does this one do?

Reply:
this test was originally here before my changes!